hass_nabucasa/ice_servers.py

```diff
@@ -20,18 +20,35 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class NabucasaIceServer(RTCIceServer):
+    """ICE server for Nabucasa."""
+
+    expiration_timestamp: int | None = None
+
+    def __init__(
+        self,
+        urls: list[str],
+        username: str | None = None,
+        credential: str | None = None,
+        expiration_timestamp: int | None = None,
+    ) -> None:
+        """Initialize Nabucasa ICE server."""
+        super().__init__(urls, username, credential)
+        self.expiration_timestamp = expiration_timestamp
```

Comment:
```suggestion
@dataclass
class NabucasaIceServer(RTCIceServer):
    """ICE server for Nabucasa."""

    expiration_timestamp: int | None = None
```
Would this not be enough?
You need to also add this import:
```py
from dataclasses import dataclass
```